metaseq/launcher/opt_baselines.py

```diff
@@ -90,7 +90,13 @@ def get_grid(args):
         if os.path.exists(args.data):
             DATA_ROOT = data_loc_by_env
         else:
-            raise RuntimeError("Where are you running this?! Check DATA_LOCATIONS.")
+            raise RuntimeError(
+                "Where are you running this?! Check DATA_LOCATIONS or pass --data "
+                "pointing to a directory with 'data' and 'tokenizers' folders."
+            )
+    elif not args.benchmark:
```

Comment:
Nit: move this conditional to top-level to make this branching easier to read? Something like:

```
if not args.benchmark:
    if args.data is None:
    ...
    else:
    ...
```